client/src/Views/Reports/MonthRangePicker.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+  <v-row class="row-container">
+    <v-col cols="11" sm="3">
+      <v-menu
```

Comment:
I added a reminder to come back to this on it being menu or dialogue, as well as having both open at once